### PR TITLE
Features/sharing

### PR DESCRIPTION
**Pull request by jeredepp**
Added non-tracking share links for every article

### CODE REVIEW

File: source/layouts/article.haml [hunk 1]
```diff
@@ -15,3 +15,10 @@
 
       %p
         = link_to '&laquo; Zurück zur Übersicht', '/', class: 'back'
+
+      %p{:class => "share"}
+        Share on
+        = link_to 'Facebook', "https://www.facebook.com/sharer/sharer.php?u=#{config[:page_url]}#{current_page.url}"
+        = link_to 'Twitter', "https://twitter.com/intent/tweet?text=#{config[:page_url]}#{current_page.url}"
+        = link_to 'LinkedIn', "https://www.linkedin.com/cws/share?url=#{config[:page_url]}#{current_page.url}"
+        = link_to 'Email', "mailto:aFriend@example.com?&subject=#{config[:page_url]}#{current_page.url}"
```
Comment (koffeinfrei) [inline source/layouts/article.haml]: * I'd rather use the article's title as the email subject
* Better use `current_article.url`
* To be safe about the url concatenation use `File.join`
* I'd omit the sample email. The `link_to` (or `mail_to`) helper doesn't allow this, which is why I'd use a standard link

 `%a{ href: "mailto:?subject=#{current_article.title}&body=#{File.join(config[:page_url], current_page.url)}" } Email`

File: source/layouts/article.haml [hunk 2]
```diff
@@ -15,3 +15,10 @@
 
       %p
         = link_to '&laquo; Zurück zur Übersicht', '/', class: 'back'
+
+      %p{:class => "share"}
+        Share on
+        = link_to 'Facebook', "https://www.facebook.com/sharer/sharer.php?u=#{config[:page_url]}#{current_page.url}"
```
Comment (koffeinfrei) [inline source/layouts/article.haml]: How about adding a helper method for the url? You can put it in `config.rb`.

File: source/layouts/article.haml (hunk 1 above)
Comment (jeredepp) [inline source/layouts/article.haml]: done

File: source/layouts/article.haml (hunk 2 above)
Comment (jeredepp) [inline source/layouts/article.haml]: helper method done

File: config.rb [hunk 3]
```diff
@@ -105,4 +104,8 @@
   def article_author(article)
     data.authors[article.data.author]['name']
   end
+
+  def get_full_post_url
+    File.join(config[:page_url],current_article.url)
```
Comment (koffeinfrei) [inline config.rb]: missing space after comma :policeman:

File: config.rb [hunk 4]
```diff
@@ -105,4 +104,8 @@
   def article_author(article)
     data.authors[article.data.author]['name']
   end
+
+  def get_full_post_url
```
Comment (koffeinfrei) [inline config.rb]: maybe `full_article_url` to keep it consistent with the method above?

and i'd pass the `article` as parameter.

Comment (jeredepp) [inline config.rb]: Renamed the function for great consistency and added `article` as param

File: config.rb (hunk 3 above)
Comment (jeredepp) [inline config.rb]: Sorry captain

File: config.rb (hunk 4 above)
Comment (koffeinfrei) [inline config.rb]: :bowtie: happy

File: config.rb (hunk 3 above)
Comment (koffeinfrei) [inline config.rb]: :policeman: happy

**Pull request merged**